app/src/main/java/com/example/fbufinalapp/EditDestinationActivity.java

```diff
@@ -183,7 +189,7 @@ public void onClick(View v) {
                     e.printStackTrace();
                 }
                 destination.setIsDay(false);
-                destination.setName(String.valueOf(binding.tvLocation.getText()));
+                destination.setName(String.valueOf(binding.etDestName.getText()));
```

Comment:
What type does `getText()` return?

Reply:
It returns an Editable object, so unfortunately I have to cast it to a String when I want to use it :)